packages/fuels/Cargo.toml

```diff
@@ -37,4 +37,5 @@ std = [
     "fuels-test-helpers/std",
     "fuels-types/std",
 ]
+test-type-paths = []
```

Comment:
Could you remind me again why we need this?

Reply:
The e2e tests for type paths don't compile unless the sway projects used in them are themselves compiled with the type path forc feature flag.

Using #ignore on the test doesn't help since they still get compiled.

Only practical way was to signal via this flag when the sway projects are compiled with the type paths flag.

So, when our ci uses all features it will use this one as well and test type paths taking care to download the correct sway artifacts.

When testing without the feature the test is going to get disabled and the normal artifacts are going to get downloaded.

This way we should catch regressions.

I couldn't find a flag that will be usable only when running tests. 

It has no effect on the sdk code.

Reply:
Ah, that makes sense. Then this will be used only while the type path feature is feature-flagged in `forc`, right? We should add a `TODO` comment here so we can easily find and remove this once the flag is gone!